csharp/advanced/Bridge.Codemasters/Bridge.Codemasters.Quix/Bridge.Codemasters.Quix.csproj

```diff
@@ -6,7 +6,7 @@
 
     <ItemGroup>
         <PackageReference Include="Quix.Sdk.Logging" Version="0.4.10-20230127.2"/>
```

Comment:
Logging

Reply:
Removed all references of Quix.Sdk.Logging package.